src/identity/storage/WrappedFetchAdapterFactory.ts

```diff
@@ -17,51 +21,95 @@ import namedNode = DataFactory.namedNode;
 export class WrappedFetchAdapter implements Adapter {
```

Comment:
Does this class have the right name?

Reply:
No it doesn't. I probably renamed it like this in the big rename PR and didn't fully comprehend the behaviour yet.

Going to rename it to `WebIdAdapterFactory`.